From: David Edmundson <david@davidedmundson.co.uk>
Date: Wed, 14 Oct 2015 16:25:28 +0000
Subject: Don't assume dbus-launch autolaunch is still broken
X-Git-Url: http://quickgit.kde.org/?p=plasma-workspace.git&a=commitdiff&h=f949f161a77114203d0489b2020011be019b47d0
---
Don't assume dbus-launch autolaunch is still broken

1) it's not broken, that comment is ancient

2) the test is bust. DBus path can also be stored as an x property.

Before this patch the situation is worse as potentially we risk
launching two dbus daemons for the same session.

BUG: 352251
REVIEW: 125637
---


--- a/startkde/startkde.cmake
+++ b/startkde/startkde.cmake
@@ -283,10 +283,6 @@
 export XDG_DATA_DIRS
 
 # Make sure that D-Bus is running
-# D-Bus autolaunch is broken
-if test -z "$DBUS_SESSION_BUS_ADDRESS" ; then
-    eval `dbus-launch --sh-syntax --exit-with-session`
-fi
 if $qdbus >/dev/null 2>/dev/null; then
     : # ok
 else

--- a/startkde/startplasmacompositor.cmake
+++ b/startkde/startplasmacompositor.cmake
@@ -160,10 +160,6 @@
 export XDG_DATA_DIRS
 
 # Make sure that D-Bus is running
-# D-Bus autolaunch is broken
-if test -z "$DBUS_SESSION_BUS_ADDRESS" ; then
-    eval `dbus-launch --sh-syntax --exit-with-session`
-fi
 if $qdbus >/dev/null 2>/dev/null; then
     : # ok
 else

